src/wolfgame.py

```diff
@@ -673,12 +673,19 @@ def join_timer_handler(var):
             for num in var.PING_IF_NUMS_ACCS:
                 if num <= len(pl):
                     for acc in var.PING_IF_NUMS_ACCS[num]:
+                        temp = users._get(get_nick(acc))
```

Comment:
1. call the variable something other than temp (perhaps "user")
2. eliminate the get_nick call: user = users._get(account=acc)